libraries/common/src/main/java/androidx/media3/common/util/BundleUtil.java

```diff
@@ -1,112 +0,0 @@
-/*
- * Copyright 2021 The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package androidx.media3.common.util;
-
-import android.os.Bundle;
-import android.os.IBinder;
-import androidx.annotation.Nullable;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-
-/** Utilities for {@link Bundle}. */
-@UnstableApi
```

Comment:
Since this class is marked as `@UnstableApi`, I think it's safe to delete it. If you prefer to deprecate it first, let me know, and I'll update my PR.